src/sentry/auth/providers/saml2.py

```diff
@@ -14,7 +14,9 @@
 
 try:
     from onelogin.saml2.auth import OneLogin_Saml2_Auth, OneLogin_Saml2_Settings
+    HAS_SAML2 = True
 except ImportError:
+    HAS_SAML2 = False
 
     def OneLogin_Saml2_Auth(*args, **kwargs):
```

Comment:
I think that's fine, but the indentation here is only these two stub functions. That way the rest of the module works fine. It was just bare minimum way to handle the dependency being optionally installed without blowing up the world.